packages/storage/amplify_storage_s3/pubspec.yaml

```
@@ -10,26 +10,15 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
-  amplify_storage_s3_android: 1.0.0-next.0
-  amplify_storage_s3_ios: 1.0.0-next.0
   amplify_core: 1.0.0-next.0
```

Let's use semver in the new packages

```suggestion
  amplify_core: ^1.0.0-next.0
```